### PR TITLE
Update AWS Private Link security group port guide

### PR DESCRIPTION
**Pull request by simonlord**

### CODE REVIEW

File: modules/networking/pages/aws-privatelink.adoc [hunk 1]
```diff
@@ -286,18 +289,46 @@ aws ec2 authorize-security-group-ingress --region $REGION --profile $PROFILE \
...
+    --protocol "tcp" \
+    --port 32192-32194 \
+    --cidr 0.0.0.0/0
+aws ec2 authorize-security-group-ingress --region $REGION --profile $PROFILE \
+    --group-id $SECURITY_GROUP_ID \
+    --protocol "tcp" \
+    --port 32292-32294 \
+    --cidr 0.0.0.0/0
+# Redpanda Panda Proxy port ranges
+#   There are 3 ranges that alternate during normal system maintenance and all 3 must be open
```
Comment (asimms41) [inline modules/networking/pages/aws-privatelink.adoc:313]:
```suggestion
#   There are three port ranges that alternate during normal system maintenance, and all three must be open. Adjust the end port number if the Redpanda broker count is not three.
```

**Review by asimms41 — CHANGES_REQUESTED** [review]
Hi Simon, Looks good. Just a couple of suggestions.

**Comment by simonlord** [comment]
@asimms41 I accepted your changes, thanks. Ready for review again

**Review by paulzhang97 — APPROVED** [review]

File: modules/networking/pages/aws-privatelink.adoc [hunk 2]
```diff
@@ -286,18 +289,40 @@ aws ec2 authorize-security-group-ingress --region $REGION --profile $PROFILE \
     --protocol "tcp" \
     --port 443-443 \
     --cidr 0.0.0.0/0
-# Adjust the port 32094 if the Redpanda broker count is not 3.
+# Redpanda Kafka broker port ranges
+#   There are three broker port ranges that alternate during normal system maintenance, and all three must be open.  Adjust the end port number if the Redpanda broker count is not three.
```
Comment (asimms41) [inline modules/networking/pages/aws-privatelink.adoc:293]:
```suggestion
#   There are three broker port ranges that alternate during normal system maintenance, and all three must be open. Adjust the end port number if the Redpanda broker count is not three.
```

**Review by asimms41 — CHANGES_REQUESTED** [review]
Just one very small change then good to go.

**Comment by simonlord** [comment]
@asimms41 I accepted your change, thanks. Ready for review again

**Review by asimms41 — APPROVED** [review]
LGTM

**Pull request merged**